client/extensions/woocommerce/state/sites/settings/stripe-connect-account/selectors.js

```diff
@@ -22,11 +22,15 @@ const getRawSettings = ( state, siteId ) => {
 	] );
 };
 
+export function getIsDisconnecting( state, siteId = getSelectedSiteId( state ) ) {
```

Comment:
maybe documentation

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/19244/commits/2d81a7a15a8b02918fd6b2ef39b9a54500e0b52c